### DISCUSSION TITLE
How to output solver stats with codegen

### DISCUSSION THREAD

**Discussion by RickDW** [casadi/casadi#3203] [Q&A] [answered]
Hi folks, I've implemented an S-function in Simulink using Casadi's code generator ([from Casadi' s blog](https://web.casadi.org/blog/s-function/)). Now I'm wondering whether it's possible to output the statistics of an NLP solver and to display them in Simulink. Is this supported?

**Comment by jgillis** [comment, marked as answer]
As of 3.6.3 there is no such support, unfortunately. Ideas for implementation are here https://github.com/casadi/casadi/issues/2106 ..
